_Tests/AudibleApi.Tests/appsettings.json

```diff
@@ -1,4 +1,4 @@
 {
   "//": "this file MUST go in the bin root. Not in a subdir like L1 even though that's the only place which should use this",
-  "LibationFiles": "\\"
+  "LibationFiles": ".\\"
```

Comment:
Innovative usage. For me, I meant \\ as a placeholder for a file path. Do you copy your appsettings file into the same dir?

Reply:
I did. I found that it was already treating "\\" as a path and trying to locate AppSettings.Json at C:\